person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    return true if age >= 18 || @parent_permission
+
+    false
+  end
```

Comment:
- [ ] For best practices, you can refactor this method above to something like this:
```ruby
def can_use_services? 
    of_age?() || @parent_permission
end
```